tck/rest/src/main/java/org/eclipse/microprofile/metrics/test/ReusedMetricsTest.java

```diff
@@ -142,20 +144,18 @@ public void setB() {
     @InSequence(4)
     public void testSharedCounterAgain() {
 
-        Header acceptJson = new Header("Accept", APPLICATION_JSON);
-
-        Response resp = given().header(acceptJson).get("/metrics/application");
-        JsonPath filteredJSONPath =
-                new JsonPath(resp.jsonPath().prettify().replaceAll(JSON_APP_LABEL_REGEX, JSON_APP_LABEL_REGEXS_SUB));
+        Response resp = given().header("Accept", TEXT_PLAIN).get("/metrics?scope=application");
         ResponseBuilder responseBuilder = new ResponseBuilder();
         responseBuilder.clone(resp);
-        responseBuilder.setBody(filteredJSONPath.prettify());
+        responseBuilder.setBody(filterOutAppLabelPromMetrics(resp.getBody().asString()));
         resp = responseBuilder.build();
 
-        resp.then()
-                .assertThat().body("'countMe2;tier=integration'", equalTo(2))
-                .assertThat().body("'timeMe2'.'count;tier=integration'", equalTo(2));
-
+        resp.then().statusCode(200)
+                .and()
+                .body(containsString("# TYPE countMe2_total counter"))
```

Comment:
is the addition of "_total" something that's common across prometheus clients?

Reply:
I believe that's as per the Prometheus exposition format. The Meric name in the help and type must match the metric name.
https://prometheus.io/docs/instrumenting/exposition_formats/#comments-help-text-and-type-information